src/Autocomplete/src/Form/AutocompleteChoiceTypeExtension.php

```diff
@@ -91,6 +95,7 @@ public function configureOptions(OptionsResolver $resolver)
             'allow_options_create' => false,
             'no_results_found_text' => 'No results found',
             'no_more_results_text' => 'No more results',
+            'min_characters' => 3,
             'max_results' => 10,
         ]);
```

Comment:
To test this, you could:

A) Pass the `min_character` option to one of the fields here https://github.com/symfony/ux/blob/2.x/src/Autocomplete/tests/Fixtures/Form/ProductType.php

B) Assert that the proper value attribute is rendered here: https://github.com/symfony/ux/blob/bb10dcff9dc9dc06fb023cf046bfcf9cc14beb5c/src/Autocomplete/tests/Functional/AutocompleteFormRenderingTest.php#L28-L40